var/spack/repos/builtin/packages/kakoune/package.py

```diff
@@ -16,13 +16,18 @@ class Kakoune(MakefilePackage):
 
     maintainers("Bambi")
 
+    version(
+        "2023.08.05", sha256="3e45151e0addd3500de2d6a29b5aacf2267c42bb256d44a782e73defb29cda5c"
+    )
     version(
         "2021.11.08", sha256="aa30889d9da11331a243a8f40fe4f6a8619321b19217debac8f565e06eddb5f4"
     )
 
     depends_on("ncurses")
 
     conflicts("%gcc@:8", when="@2021.11.08", msg="GCC version must be at least 9.0!")
+    conflicts("%gcc@:10.3", when="@2023.08.05", msg="GCC version must be at least 10.3!")
+    conflicts("%llvm@:11", when="@2023.08.05", msg="Clang version must be at least 11!")
```

Comment:
Ranges include the upper bound:
```suggestion
    conflicts("%gcc@:10.2", when="@2023.08.05", msg="GCC version must be at least 10.3!")
    conflicts("%llvm@:10", when="@2023.08.05", msg="Clang version must be at least 11!")
```